PDFUnblocker.py

```
@@ -0,0 +1,63 @@
+import os
+from pathlib import Path
```

Ruff

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)